packaging/datadog-agent/source/install_agent.sh

```diff
@@ -40,6 +40,12 @@ if [ -n "$DD_API_KEY" ]; then
     apikey=$DD_API_KEY
 fi
 
+if [ -n "$DD_INSTALL_ONLY" ] && $DD_INSTALL_ONLY ; then
```

Comment:
I don't like the `& $DD_INSTALL_ONLY`, because if you put `DD_INSTALL_ONLY=True`, it won't work.
Probably just checking that the env var is here is enough. (or else checking that it's different than `false`)